### DISCUSSION TITLE
Hope to support .net9

### DISCUSSION THREAD

**Discussion by ZhaoYis** [CommunityToolkit/dotnet#995] [Ideas]
<img width="1400" alt="image" src="https://github.com/user-attachments/assets/dd795a1f-0e8c-47e3-ae83-d074d3a95fb4">

**Comment by echoix** [comment]
The C# language version and dotnet versions aren't the same. Dotnet 9 that was recently released introduced C# 13 support, so you'd still have a problem. I see there was a second preview release though.

**Reply by ZhaoYis** [reply]
ok.3ks😊
